src/main/java/gift/auth/LoginMemberArgumentResolver.java

```diff
@@ -0,0 +1,44 @@
+package gift.auth;
+
+import gift.entity.Member;
+import gift.service.MemberService;
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.core.MethodParameter;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.support.WebDataBinderFactory;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.ModelAndViewContainer;
+
+@Component
+public class LoginMemberArgumentResolver implements HandlerMethodArgumentResolver {
+
+    private final MemberService memberService;
+
+    public LoginMemberArgumentResolver(MemberService memberService) {
+        this.memberService = memberService;
+    }
+
+    @Override
+    public boolean supportsParameter(MethodParameter parameter) {
+        return parameter.hasParameterAnnotation(LoginMember.class);
+    }
+
+    @Override
+    public Object resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer,
+                                  NativeWebRequest webRequest, WebDataBinderFactory binderFactory) {
+        HttpServletRequest request = webRequest.getNativeRequest(HttpServletRequest.class);
+        String authorization = request.getHeader("Authorization");
```

Comment:
`Authorization`을 하드코딩하기보다는 이미 구현된 HttpHeaders의 상수를 사용하면 더 실수할 여지를 줄일 수 있을 것 같아요. 😃 

```suggestion
        String authorization = request.getHeader(HttpHeaders.AUTHORIZATION);
```